src/ophys_etl/workflows/pipeline_modules/trace_processing/event_detection.py

```diff
@@ -0,0 +1,72 @@
+from types import ModuleType
+from typing import List
+
+from ophys_etl.modules import event_detection
+from ophys_etl.workflows.ophys_experiment import OphysExperiment
+from ophys_etl.workflows.output_file import OutputFile
+from ophys_etl.workflows.pipeline_module import PipelineModule
+from ophys_etl.workflows.workflow_steps import WorkflowStepEnum
+from ophys_etl.workflows.well_known_file_types import WellKnownFileTypeEnum
+
+
+class EventDetection(PipelineModule):
+    """Wrapper the event detection module"""
+
+    def __init__(
+        self,
+        ophys_experiment: OphysExperiment,
+        prevent_file_overwrites: bool = True,
+        **kwargs
+    ):
+        super().__init__(
+            ophys_experiment=ophys_experiment,
+            prevent_file_overwrites=prevent_file_overwrites,
+            **kwargs
+        )
+
+        dff_traces: OutputFile = kwargs[
+            "dff_traces"
+        ]
+        self._dff_traces_file = str(
+            dff_traces.path
+        )
+
+    @property
+    def _executable(self) -> ModuleType:
+        return event_detection
+
+    @property
+    def queue_name(self) -> WorkflowStepEnum:
+        return WorkflowStepEnum.DEMIX_TRACES
+
+    @property
+    def inputs(self):
+        roi_ids = []
+        for roi in self.ophys_experiment.roi:
```

Comment:
that's the wrong property name

please add a test for the inputs so that this bug is caught